actioncable/Rakefile

```diff
@@ -11,3 +13,40 @@ Rake::TestTask.new do |t|
   t.verbose = true
   t.ruby_opts = ["--dev"] if defined?(JRUBY_VERSION)
 end
+
+namespace :assets do
+  task :precompile do
+    ## Check for coffee executable
+    require 'mkmf'
+    unless find_executable('coffee')
+      raise "In order to compile Action Cable's assets, you must have the "\
+        " coffee-script npm package installed."
+    end
+
+    Dir.chdir(ASSETS_ROOT) do
+      puts 'Compiling Action Cable front-end Coffee Script to JS...'
+      `coffee --output action_cable_js --compile action_cable`
+
+      ## Add requires for consumer.js
+      f = File.open("#{ASSETS_ROOT}/action_cable_js/consumer.js", "r+")
+      lines = f.readlines
+      f.close
+
+      lines = [
+        "//= require action_cable/connection\n",
+        "//= require action_cable/connection_monitor\n",
+        "//= require action_cable/subscriptions\n",
+        "//= require action_cable/subscription\n"
+      ] + lines
+
+      output = File.new("#{ASSETS_ROOT}/action_cable_js/consumer.js", "w")
+      lines.each { |line| output.write line }
+      output.close
+
+      ## Output results
+      puts 'All done!'
+      puts '**Please note:** Due to action_cable.coffee.erb having a .erb'\
+        ' extension, you will need to manually compile this.'
```

Comment:
We can fix this by pushing `ActionCable.INTERNAL` into a separate file, and just making it `.js.erb`